docs/_advanced/ja_deferring_initialization.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 初期化の延期
```

Comment:
It's already understandable! 🤣 but @wongjas and I can suggest updates once we finalize the English version